AntennaTracker/system.cpp

```diff
@@ -250,23 +250,10 @@ bool Tracker::should_log(uint32_t mask)
 }
 
 
-#include <AP_Camera/AP_Camera.h>
 #include <AP_AdvancedFailsafe/AP_AdvancedFailsafe.h>
 #include <AP_Avoidance/AP_Avoidance.h>
 #include <AP_ADSB/AP_ADSB.h>
 
-/* dummy methods to avoid having to link against AP_Camera */
-void AP_Camera::control_msg(const mavlink_message_t &) {}
-void AP_Camera::configure(float, float, float, float, float, float, float) {}
-void AP_Camera::control(float, float, float, float, float, float) {}
-void AP_Camera::send_feedback(mavlink_channel_t chan) {}
-void AP_Camera::take_picture() {}
-namespace AP {
-    AP_Camera *camera() {
-        return nullptr;
-    }
-};
-
 /* end dummy methods to avoid having to link against AP_Camera */
```

Comment:
this line need to be remove too
@peterbarker could you confirm that it is fine to you to remove those ?

Reply:
See my separate PR.